Packaging/windows/dos-prep.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+apt-get update
+apt-get install bison flex curl gcc g++ make texinfo zlib1g-dev tar bzip2 gzip xz-utils unzip python3-dev m4 dos2unix nasm
+apt-get install cmake
```

Comment:
This should have `sudo`. If the whole script is run as `sudo`, it installs to /root/local.
Alternatively, we could have the whole thing run as root and install to `/opt/i386-pc-msdosdjgpp-toolchain`

Reply:
I've pushed a commit that cleans up the build script (installs to opt, handles sudo correctly). You'll need to rerun dos-prep.sh